controllers/nodeagent.go

```diff
@@ -330,6 +331,18 @@ func (r *DPAReconciler) customizeNodeAgentDaemonset(dpa *oadpv1alpha1.DataProtec
 			Privileged: pointer.Bool(true),
 		}
 
+		// append node agent server args if specified via UnsupportedNodeAgentArgs
+		if len(dpa.Spec.Configuration.NodeAgent.UnsupportedNodeAgentArgs) > 0 {
+			unsupportedArgs := strings.TrimSpace(dpa.Spec.Configuration.NodeAgent.UnsupportedNodeAgentArgs)
+			unsupportedArgsList := strings.Split(unsupportedArgs, ",")
```

Comment:
would be interesting to see how this would work with a flag like following
```
velero server \
--restore-resource-priorities=customresourcedefinitions,namespaces,storageclasses,\
volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,\
volumesnapshots.snapshot.storage.k8s.io,persistentvolumes,persistentvolumeclaims,secrets,\
configmaps,serviceaccounts,limitranges,pods,replicasets.apps,clusters.cluster.x-k8s.io,\
clusterresourcesets.addons.cluster.x-k8s.io
```

Reply:
Hmm, should not matter, `--restore-resource-priorities` is a velero server flag. This PR is passing args to node-agent server. Let me know if I am missing something here.

Reply:
@kaovilai worry is about a flag being `stringArray`, right?

My worry reading was that instead of node-agent, daemonSet would run velero server command. But as it is coded, end command would be `velero node-agent server velero server --restore-resource-priorities=...`, and velero CLI should take care of validation, right?

Reply:
today, I think `node-agent server` has one flag that is `stringArray`
```
--features stringArray             Comma-separated list of features to enable for this Velero process. Combines with values from $HOME/.config/velero/config.json if present
```

Is it possible to user to pass more then one value to it using UnsupportedNodeAgentArgs?

Reply:
this could also hurt us in current implementation
```
--vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
```
(command to try to find comma problems: `go run cmd/velero/velero.go help node-agent server | grep -i comma`)

Reply:
we do not use velero CLI for installation @mateusoliveira43

Reply:
is not the binary run by daemonSet containers velero CLI?

Reply:
yes but the container struct is built by us including the args that are passed along

Reply:
> This PR is passing args to node-agent server. Let me know if I am missing something here.

The concern is the consistency we have between node-agent and velero server. If we want to have both work, then they both should be supported the same way, including flags that accept CSV params.